Jenkinsfile

```diff
@@ -0,0 +1,26 @@
+//
```

Comment:
Is there another corresponding image in ci-build-images? I remember something about a GitHub-tools image that was going to be added there. Is this the same thing?

Reply:
No corresponding image in `ci-build-images`, my thinking is that this repo can be responsible for publishing the docker image to Nexus.  Thus eliminate the need from having to add something to `ci-build-images`.

Reply:
@ernestojeda will referencing `mavenSettings` from a different repo work as I have below?

Reply:
Yes, that works fine.